README.md

```diff
@@ -133,6 +133,7 @@ See also [Static Site Generators](#static-site-generators), [Content Management
 
   * [Anchor CMS](https://anchorcms.com/) - Anchor is a free, lightweight, faster-than-a-bullet, simple blogging system, made for art–directed posts. ([Source Code](https://github.com/anchorcms/anchor-cms)) `GPLv3` `PHP`
   * [Antville](https://antville.org) - Antville is a free, open source project aimed at the development of a high performance, feature rich weblog hosting software. ([Source Code](https://github.com/antville/antville)) `Apache` `Javascript`
+  * [BlogEngine.NET](http://dotnetblogengine.net/) - A free and open source blogging platform built with .NET that is focused on simplicity, ease of use and extendibility. ([Source Code](https://github.com/rxtur/BlogEngine.NET)) `Ms-RL` `ASP.NET`
```

Comment:
Can you add

    * Ms-Rl - [Microsoft Reciprocical License](https://opensource.org/licenses/MS-RL)

to the [List of licenses](https://github.com/Kickball/awesome-selfhosted#list-of-licenses) ?